app/dashboard/router.py

```diff
@@ -269,6 +270,18 @@ def get_queryset(self):
             end = start + int(limit)
             queryset = queryset[start:end]
 
+        # workaround for https://github.com/gitcoinco/web/issues/3348
+        pks = queryset.values_list('pk', flat=True)
+        queryset = Bounty.objects.filter(pk__in=pks)
+
+        # order
+        # this is applied twice.. if you change this block, 
+        # also change the block 20 lines up
+        order_by = self.request.query_params.get('order_by')
+        if order_by and order_by != 'null':
+            queryset = queryset.order_by(order_by)
+
```

Comment:
is it worth making this into a DRY function so can be reused rather than writing twice (or maybe more times in future?) ? @owocki @mbeacom

Reply:
@SaptakS I don't disagree.  We should DRY this up, but we should probably address: #623 sooner rather than later.

If we're documenting to update both, should we consider moving these off to another method and/or dropping one of them outright?

Reply:
Seems like https://stackoverflow.com/a/38390480/2582437 is exactly what is needed here but i'm not aware of performance.

Reply:
I would go for moving this off to another method.